src/components/Layout.tsx

```diff
@@ -35,7 +35,7 @@ export const Layout: FunctionComponent<LayoutProps> = ({
     <>
       {head}
       {cookieBanner}
-      <a href="#main-content" className="govuk-skip-link" role="main">
+      <a href="#main-content" className="govuk-skip-link">
```

Comment:
Caught this thru unit testing using `getByRole`, which was throwing up that there were two elements with the `role="main"`.  (See https://testing-library.com/docs/queries/about).

The `main` role is for the element that contains the meat-and-potatoes of the page, e.g. a newspaper article or blog post, not the link that points at it.